src/Symfony/Component/Form/Extension/Core/DataTransformer/DateIntervalToStringTransformer.php

```diff
@@ -94,8 +94,8 @@ public function reverseTransform($value)
         return $dateInterval;
     }
 
-    private function isISO8601(string $string)
+    private function isISO8601(string $string): bool
     {
-        return preg_match('/^P(?=\w*(?:\d|%\w))(?:\d+Y|%[yY]Y)?(?:\d+M|%[mM]M)?(?:(?:\d+D|%[dD]D)|(?:\d+W|%[wW]W))?(?:T(?:\d+H|[hH]H)?(?:\d+M|[iI]M)?(?:\d+S|[sS]S)?)?$/', $string);
+        return (bool) preg_match('/^P(?=\w*(?:\d|%\w))(?:\d+Y|%[yY]Y)?(?:\d+M|%[mM]M)?(?:(?:\d+D|%[dD]D)|(?:\d+W|%[wW]W))?(?:T(?:\d+H|[hH]H)?(?:\d+M|[iI]M)?(?:\d+S|[sS]S)?)?$/', $string);
```

Comment:
should be removed, the return type already handle the cast

Reply:
Fair enough, reverted.